src/AspNet.Security.OAuth.Xiaomi/XiaomiAuthenticationHandler.cs

```diff
@@ -101,7 +101,7 @@ protected override async Task<OAuthTokenResponse> ExchangeCodeAsync([NotNull] OA
 
         var payload = JsonDocument.Parse(json);
 
-        var errorCode = payload.RootElement.GetString("errcode");
+        var errorCode = payload.RootElement.GetString("error");
         if (!string.IsNullOrEmpty(errorCode))
         {
             Log.ExchangeCodeErrorCode(Logger, errorCode, response.Headers.ToString(), json);
```

Comment:
Let's rename the variable, the log method and update the log message template to reflect this change.